source/MaterialXShaderGen/ShaderGenerators/Common/Compare.cpp

```diff
@@ -1,4 +1,4 @@
-#include <MaterialXShaderGen/Implementations/Compare.h>
+#include "Compare.h"
```

Comment:
Although this is a more compact pattern for same-folder includes, we've occasionally run into situations where identical names exist on the client's include path.  For that reason, we usually write out the entire path for MaterialX includes, with the library name (e.g. MaterialXShaderGen) at the beginning of the path.

Reply:
That's a very good point! I will fix this. Thanks